### PR TITLE
(DOCSP-31144) changing deployment model include all pre-change implications

### PR DESCRIPTION
**Pull request by amalhotra-mdb**
## Pull Request Info

The docs below does not mention all the implications the customer needs to make before changing deployment models.

https://www.mongodb.com/docs/atlas/app-services/apps/change-deployment-models/#change-deployment-models 

In the UI it shows these things at the point which the customer goes to actually perform the change but this is too late. They need to know what to expect prior to scheduling the change on their side.

image

Please also see internal documentation regarding this.

### Jira

- https://jira.mongodb.org/browse/DOCSP-31144

### Staged Changes

- [Change Deployment Models](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-31144/)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [ ] Create Jira ticket for corresponding docs-realm update(s), if any
- [ ] Checked/updated Admin API
- [ ] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/apps/change-deployment-models.txt [hunk 1]
```diff
@@ -49,6 +50,18 @@ is in progress, the new migration does not run and fails with an error.
 Before You Begin
 ~~~~~~~~~~~~~~~~
 
+Before changing deployment models, please note the following: 
+
+- Suspended Triggers will restart.
+- Any existing Private Endpoints will need to be recreated for the new region.
+- The oplog replication window associated with any running Triggers is large
```
Comment (cbullinger) [inline source/apps/change-deployment-models.txt:57]: I don't really understand why this is included as a bullet. And is it associated with the first bullet? i.e. is a "suspended trigger" one that is restarted after being paused? 
is there a better way to phrase this so it's more obvious why I should note this?

File: source/apps/change-deployment-models.txt [hunk 2]
```diff
@@ -49,6 +50,18 @@ is in progress, the new migration does not run and fails with an error.
 Before You Begin
 ~~~~~~~~~~~~~~~~
 
+Before changing deployment models, please note the following:
```
Comment (cbullinger) [inline source/apps/change-deployment-models.txt:53]: Should we also note the loss of the logs and drafts?

```suggestion
Before changing deployment models, please note the following: 

- All logs and drafts will be lost.
```

File: source/apps/change-deployment-models.txt (hunk 1 above)
Comment (cbullinger) [reply source/apps/change-deployment-models.txt:57]:
```suggestion
- Triggers and Device Sync operations will be paused during the migration.
```
(again, is a paused trigger == suspended trigger?)

**Review by cbullinger — CHANGES_REQUESTED** [review]

File: source/apps/change-deployment-models.txt [hunk 3]
```diff
@@ -46,9 +41,38 @@ You can begin a deployment migration at any time. Only one migration may
...
+Before changing deployment models, please note the following: 
+
+- All logs and drafts will be lost.
+- Suspended Triggers will restart.
+- Any existing Private Endpoints will need to be recreated for the new region.
+- Triggers and Device Sync operations will be paused during the migration.
+
+.. important:: Deployment Migration is Permanent
+
+         Changing your deployment model is not a draft. This change
```
Comment (cbullinger) [inline source/apps/change-deployment-models.txt:72]:
```suggestion
```
looks like this line got duped

File: source/apps/change-deployment-models.txt [hunk 4]
```diff
@@ -17,19 +17,14 @@ or switch an App deployed in the Eastern USA from AWS to Azure. For a
...
+monitor the migration status, but no requests will be processed until the
 migration is complete. For more information, see :ref:`Deployment
 Migration Process <deployment-migration-process>`.
 
-If you connect to your App from a Realm SDK, you must update your SDK to
-a version that supports changing deployment models. If you change
-deployment models before upgrading, the SDK will not be able to connect
-and requests will fail.
-
 .. todo sync with ENG post https://jira.mongodb.org/browse/RCORE-1346 to determine minimum versions. Until then, users have to talk to support.
```
Comment (cbullinger) [inline source/apps/change-deployment-models.txt:28]: Since I see minimum SDK versions listed, did this get done? can we remove this TODO?

**Review by cbullinger — APPROVED** [review]
One small thing, but otherwise looks good!

File: source/apps/change-deployment-models.txt (hunk 4 above)
Comment (amalhotra-mdb) [reply source/apps/change-deployment-models.txt:28]: oops didn't notice that todo! will remove - thanks cory!

**Pull request merged**